style.css

```diff
@@ -70,30 +85,27 @@ footer {
   width: 100%;
 }
 
+.text_transform {
+  text-transform: capitalize;
+}
+
 .link_flex_container {
   width: 80%;
   list-style-type: none;
 }
 
-.logo {
+.font_size {
   font-size: 25px;
 }
 
+
 .links {
```

Comment:
same here. It's for links or for a link? Or maybe it's a link container... You know better than I do 🤷